src/script/vco-ie/main_warhammer/wh2_main_skv_clan_moulder/missions.lua

```diff
@@ -1,150 +1,150 @@
-local missions = {

```

Comment:
I really need to know what is happening here. Can you @Mjb141 give it a look? 

@OhManTFE Do not worry about this.

Reply:
I haven't got a clue @msolefonte - I'm converting to tabs before commits for all lua files.